<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-01-22T02:05:32 IST" name="Default suite" finished-at="2025-01-22T02:05:34 IST" duration-ms="1116">
    <groups>
    </groups>
    <test started-at="2025-01-22T02:05:32 IST" name="Default test" finished-at="2025-01-22T02:05:34 IST" duration-ms="1116">
      <class name="attributes.DependsOnMethodsMultipleExample2">
        <test-method signature="parentMethod1()[pri:0, instance:attributes.DependsOnMethodsMultipleExample2@3bb9a3ff]" started-at="2025-01-22T02:05:32 IST" name="parentMethod1" finished-at="2025-01-22T02:05:32 IST" duration-ms="19" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parentMethod1 -->
        <test-method signature="parentMethod2()[pri:0, instance:attributes.DependsOnMethodsMultipleExample2@3bb9a3ff]" started-at="2025-01-22T02:05:32 IST" name="parentMethod2" finished-at="2025-01-22T02:05:34 IST" duration-ms="1020" status="FAIL">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method attributes.DependsOnMethodsMultipleExample2.parentMethod2() didn't finish within the time-out 1000]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method attributes.DependsOnMethodsMultipleExample2.parentMethod2() didn't finish within the time-out 1000
at java.base/java.lang.Thread.sleep0(Native Method)
at java.base/java.lang.Thread.sleep(Thread.java:509)
at attributes.DependsOnMethodsMultipleExample2.parentMethod2(DependsOnMethodsMultipleExample2.java:19)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- parentMethod2 -->
        <test-method signature="testMethod1()[pri:0, instance:attributes.DependsOnMethodsMultipleExample2@3bb9a3ff]" started-at="2025-01-22T02:05:34 IST" name="testMethod1" finished-at="2025-01-22T02:05:34 IST" depends-on-methods="attributes.DependsOnMethodsMultipleExample2.parentMethod1, attributes.DependsOnMethodsMultipleExample2.parentMethod2" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method attributes.DependsOnMethodsMultipleExample2.testMethod1() on instance attributes.DependsOnMethodsMultipleExample2@3bb9a3ff depends on not successfully finished methods 
[attributes.DependsOnMethodsMultipleExample2.parentMethod1() on instance attributes.DependsOnMethodsMultipleExample2@3bb9a3ff
attributes.DependsOnMethodsMultipleExample2.parentMethod2() on instance attributes.DependsOnMethodsMultipleExample2@3bb9a3ff]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method attributes.DependsOnMethodsMultipleExample2.testMethod1() on instance attributes.DependsOnMethodsMultipleExample2@3bb9a3ff depends on not successfully finished methods 
[attributes.DependsOnMethodsMultipleExample2.parentMethod1() on instance attributes.DependsOnMethodsMultipleExample2@3bb9a3ff
attributes.DependsOnMethodsMultipleExample2.parentMethod2() on instance attributes.DependsOnMethodsMultipleExample2@3bb9a3ff]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testMethod1 -->
      </class> <!-- attributes.DependsOnMethodsMultipleExample2 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
